module/config.mjs

```diff
@@ -2693,7 +2695,7 @@ DND5E.statusEffects = {
   sleeping: {
     name: "EFFECT.DND5E.StatusSleeping",
     icon: "systems/dnd5e/icons/svg/statuses/sleeping.svg",
-    statuses: ["incapacitated", "prone", "unconscious"]
+    statuses: ["incapacitated", "unconscious"]
```

Comment:
I removed it, since it adds Unconscious. I felt it was more correct that only Unconscious adds it (sleeping will still add all its "children's" riders) since there are no explicit rules about Sleeping applying Prone directly.

Reply:
Oh right, I did see the code that recursively adds riders, that makes sense